### ISSUE TITLE
Terminating app due to uncaught exception 'kUIAExeceptionNoAccessibility', reason: 'Failed to enable accessiblity, kAXErrorServerNotFound'

### ISSUE THREAD

**Issue by sleekweasel** [appium/appium#6676]
## The problem

I was trying to test an iOS app on an iPhone 6 9.3 simulator. I observe the simulator start, and the application launch successfully, but Appium never seems to notice: Instruments continually crashes, and the crash logs complain about not being able to turn automation on.

If I take manual control of the simulator and enable Settings > General > Accessibility Inspector, it returns success. But I have that annoying floaty accessibility inspector window on the screen.

The same set-up (even the same session of Appium) works fine when the only thing I change is to make 9.3 into 9.2.
## Environment
- Appium version (or git revision) that exhibits the issue:

Appium v1.5.3 (REV 62d39fdb5ec180e37b10f5dd95b647001b9b02f7)
- Last Appium version that did not exhibit the issue (if applicable):

Don't know. But 1.5.2 is the same, and the difference is between 9.3 and 9.2 simulators on the same appium server instance.
- Desktop OS/version used to run Appium:

10.11.5 (15F34)
- Node.js version (unless using Appium.app|exe):

node -v
v0.12.4
- Mobile platform/version under test:

iPhone 6 9.3 (and 9.2, which works). XCode Version 7.3 (7D175)
- Real device or emulator/simulator:

Simulator
- Appium CLI or Appium.app|exe:

./node_modules/.bin/appium --session-override --log-level error:debug --log /tmp/appiumj.log --log-no-colors  -p 1235 &
## Details

With iPhone 9.2, it all works: I start the driver, see the app start, and after a while it comes back.

With iPhone 9.3, it starts the simulator and launches the application, but never returns because it fails to turn on automation, as shown in the crash logs.

It wasted a couple of days tracking this down because the error message doesn't suggest looking in the simulator logs. Be nice if it did. 
## Link to Appium logs

https://gist.github.com/sleekweasel/a0947dbbc258cc6264028a2c6c2d936c
## Code To Reproduce Issue [ Good To Have ]

This behaviour obtains even with a brand new no-code xcode ios project.

$ irb
2.1.5 :001 > require 'appium_lib'
2.1.5 :002 > d1 = Appium::Driver.new({ caps: { platformName: 'ios', platformVersion: '9.3', deviceName: 'iPhone 6', app: '/tmp/oodab.app', autoAcceptAlerts: false, bundleId: 'com.badoo.oodab', newCommandTimeout: 305, showIOSLog:false, fullReset: true }, appium_lib: { server_url: 'http://localhost:1235/wd/hub/', appium_port: 1235} })
2.1.5 :003 > d1.start_driver
-> POST session
[Instruments] Error launching instruments: Instruments crashed on startup
[Instruments] Error launching instruments: Instruments crashed on startup
